### PR TITLE
Added definition files for BluetoothClassicSerial Cordova plugin

### PR DESCRIPTION
**Pull request by tuvokki**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/cordova-plugin-bluetoothclassic-serial/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
+// Type definitions for cordova-plugin-bluetoothClassic-serial 0.9.8
```
Comment (DanielRosenwasser) [inline types/cordova-plugin-bluetoothclassic-serial/index.d.ts]: Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/cordova-plugin-bluetoothclassic-serial/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (DanielRosenwasser) [inline types/cordova-plugin-bluetoothclassic-serial/tsconfig.json]: Can you make sure `strictNullChecks` is on here?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by tuvokki** [comment]
@DanielRosenwasser fixed

File: types/cordova-plugin-bluetoothclassic-serial/tslint.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```
Comment (DanielRosenwasser) [inline types/cordova-plugin-bluetoothclassic-serial/tslint.json]: This rules list should be empty.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/cordova-plugin-bluetoothclassic-serial/tslint.json (hunk 3 above)
Comment (tuvokki) [reply types/cordova-plugin-bluetoothclassic-serial/tslint.json]: Removed the rules node.

**Comment by DanielRosenwasser** [comment]
Thank you!

**Pull request merged**